lib/ui/screens/create_post/create_post_model.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'create_post_model.freezed.dart';
+
+@freezed
+class CreatePostModel with _$CreatePostModel {
+  const factory CreatePostModel({
+    required String title,
+    required String body,
```

Comment:
**issue:** Remove both fields and create an empty model since these variables are never used.